.github/workflows/test.yml

```
@@ -0,0 +1,24 @@
+---
+name: Test
+
+on: [pull_request, push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install pytest nose coverage cached-property
+        python setup.py clean build install
+    - name: Run test
+      run: nosetests --with-coverage
```

```suggestion
        pip install pytest pytest-cov coverage cached-property
        python setup.py clean build install
    - name: Run test
      run: pytest
```
`nose` stopped working around Python 3.8.